### PR TITLE
Lots of fixes

### PR DESCRIPTION
**Pull request by nikomakela**

### CODE REVIEW

File: src/apollo/apollo.stories.tsx [hunk 1]
```diff
@@ -5,15 +5,18 @@ import { ComponentStory, ComponentMeta } from "@storybook/react";
...
+import { ConfigProvider } from "../core/configProvider/ConfigProvider";
+import { defaultConfig } from "../core/configProvider/defaultConfig";
+import { PageContent } from "./pageContent/PageContent";
+import { Navigation } from "./navigation/Navigation";
+import { Notification } from "./notification/Notification";
+import { Page } from "./page/Page";
+import { PageContentLayout } from "../core/pageContent/PageContentLayout";
+import { LanguageCodeEnum } from "../core";
 
+const cmsUri = "https://hkih.stage.geniem.io/graphql";
```
Comment (melniiv) [inline src/apollo/apollo.stories.tsx:17]: we have env variable, can we reuse it?

File: src/core/pageContent/utils.ts [hunk 2]
```diff
@@ -39,16 +40,16 @@ export function getCollectionCards(
...
     url: item.link,
     imageUrl: item.featuredImage?.node?.mediaItemUrl || defaultImageUrl,
-    ariaLabel: "",
-    className: "",
-    imageLabel: "",
-    subTitle: "",
-    text: item.lead,
-    customContent: "",
+    ariaLabel: item.title,
+    className: undefined,
```
Comment (melniiv) [inline src/core/pageContent/utils.ts:44]: why undefined?

**Review by melniiv — APPROVED** [review]
Good work! Some comments added

File: src/core/archiveSearchPage/ArchiveSearchPage.stories.tsx [hunk 3]
```diff
@@ -31,32 +40,48 @@ const navigation = (
       console.log("I should navigate");
     }}
     getPathnameForLanguage={({ slug, code }, currentLanguage) => {
-      const baseUrl = "http://localhost:3000";
```
[the remark this answers is not in the document]
Comment (nikomakela) [reply src/core/archiveSearchPage/ArchiveSearchPage.stories.tsx:34]: the baseUrl is actually also a wrong one sinc ethe story book loads it to 6006 by default. I will use the origin from the window.location instead.

File: src/apollo/apollo.stories.tsx (hunk 1 above)
Comment (nikomakela) [reply src/apollo/apollo.stories.tsx]: yup!

**Pull request merged**